cmd/homi/genesis/options.go

```diff
@@ -71,10 +71,6 @@ func makeGenesisAccount(addrs []common.Address, balance *big.Int) map[common.Add
 func Alloc(addrs []common.Address, balance *big.Int) Option {
 	return func(genesis *blockchain.Genesis) {
 		alloc := makeGenesisAccount(addrs, balance)
-		alloc[common.HexToAddress(contract.AddressBookContractAddress)] = blockchain.GenesisAccount{
```

Comment:
If I change it like this, is not the address book applicable even if it is not the service chain?

Reply:
The `Alloc()` function only used for the service chain.